docker/release/fedora27/Dockerfile

```diff
@@ -1,9 +1,9 @@
-# Docker image file that describes an fedora 25 image with PowerShell installed from Microsoft YUM Repo
+# Docker image file that describes an fedora 27 image with PowerShell installed from Microsoft YUM Repo
 
-FROM fedora:25
+FROM fedora:27
 
 ARG POWERSHELL_VERSION=6.0.0
-ARG IMAGE_NAME=microsoft/powershell:fedora25
+ARG IMAGE_NAME=microsoft/powershell:fedora26
```

Comment:
If its fedora 27, why are we calling it `fedora26`?

Reply:
Changed